cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/sinks/NGSICKANSink.java

```diff
@@ -544,20 +584,24 @@ private void persistAggregation(CKANAggregator aggregator, String service, Strin
      */
     public String buildOrgName(String fiwareService) throws CygnusBadConfiguration {
         String orgName;
-        
-        if (enableEncoding) {
-            orgName = NGSICharsets.encodeCKAN(fiwareService);
-        } else {
-            orgName = NGSIUtils.encode(fiwareService, false, true).toLowerCase(Locale.ENGLISH);
-        } // if else
+        if(!pkgByEntity) {
```

Comment:
In a simple `if-else` condition where the condition is a single boolean var. It's easier to read if the true statement is inside the `if` and the "negation" in the `else`. When you have more than one condition inside the `if` it makes sense to deny a boolean var.

So, in this case it would be better this way.

if(pkgByEntity) {
    orgName=fiwareService;
} else {
    .......
}

Reply:
Fixed in dfaf3a78e6536d794c2a5036dd62b9f739d05591